### PR TITLE
Update travis build matrix to include Ruby 2.2.3 for Rails5

### PR DESCRIPTION
**Pull request by KaraAJC**
contributing to issue #1303 
Utilized the following resources for the changes: 
- https://docs.travis-ci.com/user/languages/ruby/
- https://docs.travis-ci.com/user/customizing-the-build/
- comments and changes from #1324

### CODE REVIEW

**Comment by KaraAJC** [comment]
My build fails were based on "RAILS_VERSION=5" and in bundle install process getting the following error for ruby versions over 2.2.2: 
```
Could not find gem 'railties (~> 5.0) ruby' in any of the gem sources listed in your Gemfile 
or available on this machine.
```
Will investigate bundle gemfile and dependencies of rails 5

Also want to prevent RAILS_VERSION=5 from being tested below 2.2.2. Will go back to reviewing Build Matrix to exclude rails5 from building below 2.2.3

**Comment by bf4** [comment]
Hi @KaraAJC Thanks for this.

RAILS_VERSION=master is Rails 5 for now, until a release is tagged, so that's why RAILS_VERSION=5.0 doesn't work; it's just master.

In terms of 2.2.3, I'd ask @beauby if he thinks there's any advantage to testing 2.2.2 and 2.2.3

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ rvm:
   - 2.0.0
   - 2.1
   - 2.2.2
+  - 2.2.3
```
Comment (bf4) [inline .travis.yml]: @beauby should we just change 2.2.2 to 2.2.3?

**Comment by KaraAJC** [comment]
Ok, great! thank you for clarifying that! Once @beauby responds, I can remove the unnecessary rails version, and re submit my pull request

**Comment by bf4** [comment]
Just force push to the same branch you made the pr from

I expect he'll be okay with replacing 2.2.2 with 2.2.3.  There shouldn't be any breaking changes between them, and keeping both just makes the build more expensive

B mobile phone

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +37,8 @@ matrix:
     env: CAPTURE_STDERR=true
   - rvm: jruby-19mode
     env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false -Xcli.debug=true --debug'
+  - rvm: 2.2.3
+    env: RAILS_VERSION=master
```
Comment (bf4) [inline .travis.yml]: This can be removed.  It already tests on master

Comment (bf4) [inline .travis.yml]: (That's why there were two builds on master https://github.com/rails-api/active_model_serializers/pull/1360#commitcomment-14738951)

Comment (KaraAJC) [inline .travis.yml]: OK will remove and force push, thank you!

**Comment by bf4** [comment]
@KaraAJC Please rebase/squash to one nicely commented commit, give yourself credit in the changelog, force push, and I'll merge it

**Comment by beauby** [comment]
Hi @KaraAJC, thanks for your PR! Do you need any assistance to achieve the final steps? Other than that, it looks good to me.

**Comment by KaraAJC** [comment]
hi @beauby Yes, I got a bit stuck determining the best way to squash/rebase and make changes to the changelog. I'm looking for someone to pair with me on it, i should be able to finish it up today. thanks for checking in!

**Comment by bf4** [comment]
@KaraAJC if you want to join the [ams slack](http://amserializers.herokuapp.com/) or email me at http://benjaminfleischer.com/pair I'm happy to pair.

Some git resources I like linking to:

- http://rakeroutes.com/blog/deliberate-git/
- http://help.exercism.io/git-workflow.html

**Comment by bf4** [comment]
Or, I can just merge it and squash it myself, if you'd like

**Comment by bf4** [comment]
@KaraAJC You are merged, yay! :dancers:

**Comment by KaraAJC** [comment]
:fireworks: YAY!!

**Pull request closed without merging**